patch/kernel/archive/rockchip64-6.7/0000.patching_config.yaml

```diff
@@ -0,0 +1,36 @@
+config: # This is file 'patch/kernel/archive/meson64-6.4/0000.patching_config.yaml'
```

Comment:
Some c&p error I guess. Should be rk64 6.7

Reply:
Oops! Thanks for spotting this!

Reply:
This is my fault, the original .yaml proposal had those comments for pasting it out of context, and then somehow those kept bying c&p everywhere lol